mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1035,6 +1036,50 @@ func TestClientSideEncryptionProse(t *testing.T) {
 			})
 		}
 	})
+
+	// These tests only run when a KMS mock server is running on localhost:8000.
+	mt.RunOpts("kms tls tests", noClientOpts, func(mt *mtest.T) {
+		testcases := []struct {
+			name             string
+			runOnEnvVariable string
+			errMessage       string
+		}{
+			{
+				"invalid certificate",
+				"KMS_TLS_INVALID_CERT",
+				"expired",
+			},
+			{
+				"invalid hostname",
+				"KMS_TLS_INVALID_HOSTNAME",
+				"SANs",
```

Comment:
What is `SANs`?

Reply:
This is from the [spec](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#invalid-hostname-in-kms-certificate). It was something I was unsure about when writing, but nobody commented on. The real error message here is something like `"cannot validate certificate for 127.0.0.1 because it doesn't contain any IP SANs"` so I thought  `"SANs"` (server alternative names) was a good keyword to look for.

Reply:
Ah ok, that makes sense. Thanks for the explanation!